packages/checkbox-react/src/Checkbox.tsx

```diff
@@ -29,9 +30,10 @@ export function Checkbox({
         "jkl-checkbox--inline": inline,
         "jkl-checkbox--error": invalid,
     });
+    const [uuid] = useState(`jkl-checkbox-${nanoid(8)}`);
```

Comment:
This is really not a uuid 🤔

Reply:
Burde ikke bli flere renders, state blir initialisert med en (ikke uuid, men unik nok sannsynligvis) id, den blir aldri endret, så innholdet burde overleve eventuelle rerendenders.

Reply:
Hehe, tror "uuid" sitter litt i fingrene bare 😅 Kan gjerne kalle den noe annet (bare `id`?) for korrekthetens skyld. Tror heller ikke det skal bli ekstra rerenders med state?

Reply:
> men unik nok sannsynligvis

![Skjermbilde 2020-03-10 kl  15 49 07](https://user-images.githubusercontent.com/25739615/76324742-bd879180-62e6-11ea-862c-54cd770f4ade.png)
😁